### PR TITLE
chore: improve the ballot submission process

### LINKED ISSUE #520: UX: remove submit ballot popup
Remove the unnecesarry confirmation of "submit ballot popup"

Image

### PR DESCRIPTION
**Pull request by kittybest**
**Description**

- [x] remove the popup dialog before submit ballot
- [x] add error message in the toast error when `onVote`
- [x] add `check my ballot` button on the `BallotOverview` widget
- [x] adjust the styling of the `BallotOverview` widget

**Related Issues**
close #520

### CODE REVIEW

File: packages/interface/src/features/ballot/components/SubmitBallotButton.tsx [hunk 1]
```diff
@@ -44,36 +47,11 @@ export const SubmitBallotButton = ({ pollId }: ISubmitBallotButtonProps): JSX.El
...
-        buttonName="submit"
-        description="This is not the final submission. Once you submit your ballot, you can change it during the voting period."
-        isLoading={isLoading}
-        isOpen={ableToSubmit && isOpen}
-        size="sm"
-        title="submit your ballot"
-        onOpenChange={setOpen}
-      />
+    <Button variant={ableToSubmit ? "primary" : "disabled"} onClick={handleSubmitBallot}>
+      {ableToSubmit && !isLoading ? "submit your ballot" : "Exceed initial voice credits"}
```
Comment (crisgarner) [inline packages/interface/src/features/ballot/components/SubmitBallotButton.tsx:52]: Sometimes, the calculation is wrong and there is an issue about exceeding initial voice credits

image

Also seems like the first time the tx is triggered

image

**Review by crisgarner — CHANGES_REQUESTED** [review]

**Comment by crisgarner** [comment]
Thanks! just one comment while testing

File: packages/interface/src/features/ballot/components/SubmitBallotButton.tsx (hunk 1 above)
Comment (kittybest) [reply packages/interface/src/features/ballot/components/SubmitBallotButton.tsx]: I just updated this PR, your first problem should be fixed, but I don't really understand your second question?

Comment (crisgarner) [reply packages/interface/src/features/ballot/components/SubmitBallotButton.tsx]: Thanks. The second question was a bug, but seems like it has been solved

**Comment by crisgarner** [comment]
Thanks! With the new version, I noticed that the transaction gets stuck in the UI 
image

Even after the tx is confirmed
image

**Review by crisgarner — APPROVED** [review]

**Comment by crisgarner** [comment]
Thanks!

**Pull request merged**